### PR TITLE
cellMic: Bugfix

### PR DESCRIPTION
**Pull request by elad335**
Fixes a regression with cellMic and fxo management.

### CODE REVIEW

**Comment by elad335** [comment]
I will try to fix more regressionss like this..

**Comment by elad335** [comment]
I think that's all of it.

File: rpcs3/Emu/Cell/Modules/cellCamera.h [hunk 1]
```diff
@@ -360,6 +360,7 @@ class camera_context
 
 public:
 	void operator()();
+	void Reinit();
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/cellCamera.h]: Uh, there is a better word "reset". Also inconsistent naming style.

Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/cellCamera.h]: Uh, there is a better word "reset". Also inconsistent naming style.

**Pull request merged**